myuw/templates/handlebars/card/student_info.html

```diff
@@ -1,168 +1,208 @@
+                    <div class="col-sm-3">
+                        <h4 class="myuw-card-row-heading">Local Address</h4>
+                    </div>
+                    <div class="col-sm-9">
+                        <div class="myuw-card-row-content">
+                        {{#if local_address}}
+                        {{#with local_address}}
+                        {{#if street_line2}}
+                        <div><span class="">{{street_line2}}</span></div>
+                        {{/if}}
+                        {{#if street_line1}}
+                        <div><span class="">{{street_line1}}</span></div>
+                        {{/if}}
+                        {{#if city}}<span class="">{{city}}</span>{{/if}}{{#if state}}<span class="">, {{state}}</span>{{/if}}{{#if postal_code}} <span class="">{{postal_code}}</span>{{/if}}{{#if zip_code}} <span class="">{{zip_code}}</span>{{/if}}
+                        {{#if country}}
+                        <div><span class="">{{country}}</span></div>
+                        {{/if}}
+                        {{/with}}
+                        {{#if local_phone}}
+                        <div>Phone: <span class="">{{formatPhoneNumber local_phone}}</span></div>
+                        {{/if}}
+                        {{else}}
+                        <span class="my-text-muted">No address available</span>
+                        {{/if}}
+                        </div>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-sm-3">
+                        <h4 class="myuw-card-row-heading">Permanent Address</h4>
+                    </div>
+                    <div class="col-sm-9">
+                        <div class="myuw-card-row-content">
+                        {{#if permanent_address}}
+                        {{#with permanent_address}}
+                        {{#if street_line2}}
+                        <div><span class="">{{street_line2}}</span></div>
+                        {{/if}}
+                        {{#if street_line1}}
+                        <div><span class="">{{street_line1}}</span></div>
+                        {{/if}}
+                        {{#if city}}<span class="">{{city}}</span>{{/if}}{{#if state}}<span class="">, {{state}}</span>{{/if}}{{#if postal_code}} <span class="">{{postal_code}}</span>{{/if}}{{#if zip_code}} <span class="">{{zip_code}}</span>{{/if}}
+                        {{#if country}}
+                        <div><span class="">{{country}}</span></div>
+                        {{/if}}
+                        {{/with}}
+                        {{#if permanent_phone}}
+                        <div>Phone: <span class="">{{formatPhoneNumber permanent_phone}}</span></div>
+                        {{/if}}
+                        {{else}}
+                        <span class="my-text-muted">No address available</span>
```

Comment:
Is "my-text-muted" a legacy class or something from bootstrap? Should we create a new "myuw-muted" class in typography.less?